### ISSUE TITLE
Using shortened-llm for Instruct models

### ISSUE THREAD

**Issue by sriyachakravarthy** [Nota-NetsPresso/shortened-llm#18]
Hi! Can we use shortened-llm to prune Instruct models?

**Comment by lifelongeeek** [comment]
@sriyachakravarthy Of course. Please see discussion with similar issue in https://github.com/Nota-NetsPresso/shortened-llm/issues/16.

**Comment by sriyachakravarthy** [comment]
Thank you, @lifelongeeek ! 
Quick questions, what does --no_instruction argument do in the lora_retrain? and what should we exactly pass for --resume_from_checkpoint argument to continue fintetuning ?
